owl/SOMA-IO.owl

```diff
@@ -22,10 +22,10 @@ Declaration(Class(SOMA:Algorithm))
 Declaration(Class(SOMA:ArchiveFile))
 Declaration(Class(SOMA:ArchiveFormat))
 Declaration(Class(SOMA:ArchiveText))
-Declaration(Class(SOMA:ArtificialAgent))
```

Comment:
Was this moved or removed? Why?

Reply:
ArtificialAgent was defined twice: once in SOMA-OBJ and once in SOMA-IO. Since I was told that SOMA-IO was not used until now, I removed that one from SOMA-IO. However, the SOMA-OBJ ArtificialAgent used the SOMA-OBJ namespace in its IRI. I opened an Issue for that.